### ISSUE TITLE
Unable to use showInput: true for custom input color

### ISSUE THREAD

**Issue by KD25** [Jimdo/angular-spectrum-colorpicker#51]
As there is an option to allow input for color but after setting this showInput: true, input get displayed on color picker but it's not allowing me to type any thing. 

Please help me out on this.

**Comment by Xiphe** [comment]
Hi @KD25 

Could you please provide steps to reproduce the behaviour?

**Comment by Xiphe** [comment]
The [demo uses `showInput: true`](https://github.com/Jimdo/angular-spectrum-colorpicker/blob/master/demo/index.html#L31) and works fine for me.

maybe you could check it out and try to reproduce your issue there. (See [Initialize the source project](https://github.com/Jimdo/angular-spectrum-colorpicker#initialize-the-source-project))

**Comment by KD25** [comment]
Yes sure, i figure out the situation behind it but not got the technical reason for this.

This only reproduce when we use this color picker in Bootstrap modal pop-up nothing else. Its working fine if i use this out of Bootstrap modal pop-up and also in demo.

Please let me know if more details are required.

**Comment by Xiphe** [comment]
do you get any error in the js-console?

**Comment by KD25** [comment]
No, nothing in console but textbox just get disabled and there is no attribute on HTML like disable or readonly

**Comment by KD25** [comment]
Hi @Xiphe  please let me know if any update on this.

Thanks,

**Comment by Xiphe** [comment]
I still have no Idea how to reproduce this, could you provide a PR that illustrates the problem? maybe in form of a failing test-case? Or a plunker would help, too.
